### ISSUE TITLE
Linting doesn't work with multi-root workspaces

### ISSUE THREAD

**Issue by CodeForcer** [juanfranblanco/vscode-solidity#237]
Although it styles the code and performs the compilation, this extension doesn't seem to lint properly when using multi-root workspaces with either Solhint or Solium.

It works fine when you open a single project folder, just not multi-root

**Comment by juanfranblanco** [comment]
Yes, correct. All the extension works around a single workspace.

**Comment by sebastiantf** [comment]
It seems the linter doesn't pickup `.solhint.json` configs inside a subdir. Is it configurable in the extension settings? It would be nice to let the extension pickup `.solhint.json` inside a subdir, or configure it where to look for `.solhint.json` in a monorepo.

**Comment by iboxgithub** [comment]
same issue for me, I guess it is linked to this: https://forum.openzeppelin.com/t/trouble-importing-openzeppelin-contracts-into-vs-code/5230/7

**Comment by markokhman** [comment]
Was anybody able to solve this ever? Quite annoying behavior...

**Comment by iboxgithub** [comment]
nope

**Comment by juanfranblanco** [comment]
I am looking at it now

**Comment by iboxgithub** [comment]
nice 😄 thanks!

**Comment by juanfranblanco** [comment]
It is checked in now.

@sebastiantf  regarding subdirectories and mono repos, if it is at root the file will work as usual (single workspace), although yes with multiple workspaces the root is in each workspace, that will need copy and paste. The solution will be to have an absolute location for the config file, which will add complications in the future for web support.
